pkg/operator/starter.go

```diff
@@ -66,8 +68,21 @@ func RunOperator(ctx context.Context, kubeconfig *restclient.Config) error {
 	if err != nil {
 		klog.Warningf("unable to get owner reference (falling back to namespace): %v", err)
 	}
+
 	eventRecorder := events.NewKubeRecorder(kubeClient.CoreV1().Events(defaults.ImageRegistryOperatorNamespace), "image-registry-operator", controllerRef)
 
+	desiredVersion := status.VersionForOperatorFromEnv()
```

Comment:
Following the implementation of `VersionForOperatorFromEnv` I see that the value comes from the env var `OPERATOR_IMAGE_VERSION`, which the [regisitry operator manifest](https://github.com/openshift/cluster-image-registry-operator/blob/master/manifests/07-operator.yaml#L59-L79) does not specify.
Could you please add the `OPERATOR_IMAGE_VERSION` env var to the operator manifest yaml? Here's an example: https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/manifests/09_deployment.yaml#L60-L61. I think that OCP's building tools will replace the value with something other than the placeholder, so you can use the example as is in our manifest.